src/thirdparty/caman/instagram.js

```diff
@@ -0,0 +1,134 @@
+
+
+$(function() {
+  "use strict";
+
+  var CommandManager = brackets.getModule("command/CommandManager");
+  var Commands = brackets.getModule("command/Commands");
+  var fs = brackets.getModule("fileSystemImpl");
+  var Buffer = brackets.getModule("filesystem/impls/filer/BracketsFiler").Buffer;
+
+  function savePhoto(){
+    Caman(canvas, function () {
+      this.render(function () {
+            var image = this.toBase64();
+            var binaryDataStr = /^data:image\/png;base64,(.+)/.exec(image)[1];
+            //self.camera.savePhoto(base64ToBuffer(binaryDataStr));
+              var binary = window.atob(binaryDataStr);
```

Comment:
I did a search on its compatibility and it seems that IE 10 and onward can use it. Does this mean we should find a workaround for the method for versions before IE 10?